### PR TITLE
= tests for models

### PR DESCRIPTION
**Pull request by dimus**

### CODE REVIEW

File: resolver/src/test/scala/org/globalnames/resolver/model/NameStringIndicesSpec.scala [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        it("returns url link to the index data") {
+          whenReady(conn.run(nameStringIndices
+            .filter(_.dataSourceId === 169)
+            .map { _.url }.result)) { res =>
+            res(0).get should startWith("http://www.ubio.org")
+          }
+        }
+      }
+
+      describe("CONTEXT: no url") {
```
Comment (alexander-myltsev) [inline resolver/src/test/scala/org/globalnames/resolver/model/NameStringIndicesSpec.scala]: This test checks Slick, not resolver. I think we don't need it.

File: resolver/src/test/scala/org/globalnames/resolver/model/NameStringsSpec.scala [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    it("contains several records") {
+      whenReady(conn.run(nameStrings.size.result)) { res => res should be > 1 }
+    }
+
+    describe("#id") {
+      it("returns an id of a name string") {
+        whenReady(conn.run(nameStrings
+          .filter(_.name === "Stelletta cyathoides Burton 1926")
+          .map { _.id }.result.head)) { res =>
+          res.toString() shouldBe "5477686c-260f-5762-82f5-1737d850f943"
```
Comment (alexander-myltsev) [inline resolver/src/test/scala/org/globalnames/resolver/model/NameStringsSpec.scala]: `toString` -- redundant.

File: resolver/src/test/scala/org/globalnames/resolver/model/NameStringIndicesSpec.scala (hunk 1 above)
Comment (dimus) [inline resolver/src/test/scala/org/globalnames/resolver/model/NameStringIndicesSpec.scala]: it describes an "edge" case --- tests as documentation

File: .gitignore [hunk 3]
```diff
@@ -38,6 +38,9 @@ logs/
 tmp/
 VARS_PROD*.env
 pgpass*
+*.dump
```
Comment (dimus) [inline .gitignore]: lets look why we have dump generated inside of the codebase for tests

File: resolver/src/test/scala/org/globalnames/resolver/model/DataSourcesSpec.scala [hunk 4]
```diff
@@ -24,24 +21,24 @@ class DataSourcesSpec extends SpecConfig {
 
     describe("#id") {
       it("returns an id of a data source") {
-        whenReady(conn.run(dataSources.take(1).map { _.id }.result)) { res =>
-          res(0) shouldBe 1
+        whenReady(conn.run(dataSources.map { _.id }.result.head)) { res =>
+          res shouldBe an[Integer]
```
Comment (alexander-myltsev) [inline resolver/src/test/scala/org/globalnames/resolver/model/DataSourcesSpec.scala]: Looks cool and where it should be. Good catch!

**Pull request merged**